### ISSUE TITLE
introduce <aside> element for /blog

### ISSUE THREAD

**Issue by lenareinhard** [hoodiehq/hood.ie#114]
I couldn't find a way to implement an <aside> element on /blog. 
What I need: one picture (like the one on /about) embedded on the top left of each blog page (or at least on /blog).
If possible, we could also think about embededing the same picture in each blog post then, but that's only a nice-to-have.

**Comment by verpixelt** [comment]
We'll take care of this at the weekend, together.

**Comment by zoepage** [comment]
So, I guess we can close this?

**Comment by lenareinhard** [comment]
no, didn't work out as planned, the element still isn't in the blog.

**Comment by zoepage** [comment]
@verpixelt as you two are both based in Berlin, you might like to take care of it. If not, we can close and re-open, when you have time.

**Comment by verpixelt** [comment]
Where's the reason in closing and reopening the issue? It will never get fixed if we close it.

**Comment by zoepage** [comment]
I think, if this is important, it will come back again.
This is why closing makes sense for me, when no one has time to take care of it.

**Comment by lenareinhard** [comment]
The general idea behind the website concept was to have <aside> elements for all major sections which also get their own animal (see also #62) and then contain the animal png. If #62 happens, #114 will have to happen as well. If there's no more (time for more) animals, all these issues can be closed.
